_pages/topics/git_recovery.md

```diff
@@ -21,6 +21,9 @@ Rosszul írtad le a commitodat, vagy esetleg rossz változtatások kerültek bel
 ### Megoldás
 A `git commit --amend` lecseréli a `HEAD` commitot, így ahelyett hogy lenne egy "add cool feature" commit és egyből utána egy "fix embarrassing typo in previous change" commit, csak egy "add cool feature" commit van typo nélkül.  A parancs amúgy ugyanúgy működik mint mindig, tehát a `git add` vagy `git commit --patch` hívása ugyanúgy szükséges a változtatások hozzáadásához, de ha csak a commit leírását akarod változtatni akkor nem.
 
+### Git merge
```

Comment:
Another problem with the placement of the header is that it breaks the document structure.  This section is about specific problems and their solutions, not about describing basics.  So it should be moved to the section about basics.

But also this is such a shallow explanation of merges that it is better to move it to the git basics document instead, since it is not specific to troubleshooting.